src/main/kotlin/uk/gov/dluhc/printapi/service/aed/AnonymousElectorDocumentService.kt

```diff
@@ -100,4 +108,8 @@ class AnonymousElectorDocumentService(
         val contents = pdfFactory.createPdfContents(templateDetails)
         return PdfFile(PDF_FILE_NAME.format(certificateNumber), contents)
     }
+
+    private fun valueHasChanged(newValue: String?, existingValue: String?): Boolean {
+        return !newValue.isNullOrBlank() && newValue != existingValue
```

Comment:
1) What if user is trying to set the email to `null/blank` and phone to some sensible value? 
This logic might not update/blank out the email for latest AED

2) Could support that by adding a unit test in service